prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/channel/ChannelAddCommand.java

```diff
@@ -43,36 +45,55 @@ public class ChannelAddCommand extends AbstractCommand {
     @CommandLine.Option(names = CliConstants.CHANNEL_NAME, required = true)
     private String channelName;
 
-    @CommandLine.Option(names = CliConstants.CHANNEL_MANIFEST, required = true)
-    private String gavUrlOrPath;
+    @CommandLine.ArgGroup(exclusive = false)
+    ChannelGroup channelGroup;
+
+    static class ChannelGroup {
+        @CommandLine.Option(names = CliConstants.CHANNEL_MANIFEST)
+        private String manifestLocation;
+
+        @CommandLine.Option(names = CliConstants.REPOSITORIES, split = ",", paramLabel = CliConstants.REPO_URL)
+        private List<String> repositoryDefs;
+    }
+    @CommandLine.Option(names = CliConstants.CHANNEL)
```

Comment:
The default `--channel` parameter description (prospero channel add --help) doesn't work for this, we need to change it. See UsageMessages.properties, you'll need to add keys like `${prospero.dist.name}.channel.add.channel.0`, `${prospero.dist.name}.channel.add.channel.1`, `${prospero.dist.name}.channel.add.channel.2`